docs/content/concepts/logging.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: "Logging | Dagster Docs"
+description: "Dagster includes a rich and extensible logging system"
+---
+
+# Logging
+
+Dagster supports a variety of built-in logging options, as well as the ability to extend and customize them. By default, Dagster automatically tracks and captures all execution events, such as those during an asset materialization. During these events, logs like the following are generated:
+
+```bash
```

Comment:
These are all structured logs. An example including structured logs and compute logs would be great and identifying the difference would be great.